.circleci/config.yml

```diff
@@ -298,13 +298,16 @@ commands:
 
 jobs:
   secrets-check:
-    executor: machine-executor
+    executor: docker-executor
```

Comment:
Changed this to the docker-executor so the trufflehog script can have access to `python`.